components/engine_traits/src/memory_engine.rs

```diff
@@ -0,0 +1,12 @@
+// Copyright 2023 TiKV Project Authors. Licensed under Apache-2.0.
+
+use std::fmt::Debug;
+
+use crate::{Iterable, WriteBatchExt};
+
+/// A memory enigne works as a region cache caching some regions to improve the
```

Comment:
caching some regions' kv data

please descirbe how it can improve the read peformance.

Reply:
I plan to describe it in the specific struct implementing it.